dbtmetabase/dbt.py

```diff
@@ -36,12 +36,21 @@ def read_models(self, includes=[], excludes=[]) -> list:
         mb_models = []
 
         for path in (Path(self.project_path) / "models").rglob("*.yml"):
+            print(path)
             with open(path, "r") as stream:
                 schema = yaml.safe_load(stream)
+                if schema is None:
```

Comment:
When does this happen? If this is expected, there should be a warning log noting that the file is skipped.

Reply:
It happens when you have a yml file without models or sources in models directory.
This happens if you have the file structure for your project set up including yml files but have yet to populate some. 

Your right I'll add logging here.